### PR TITLE
Fix policy template

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR makes some policy template fixes/adjustments.

The updated policy template is the most used configuration. So it makes sense to use it as the default. 

Related to Jira-Issue: https://4teamwork.atlassian.net/browse/GEVER-79

### CODE REVIEW

File: opengever/policytemplates/policy_template/opengever.+package.name+/deployment-+base.server_name+-+base.deployment_number+-+base.domain+.cfg.bob [hunk 1]
```diff
@@ -19,6 +19,15 @@ solr-core-name = {{{package.name}}}
...
 develop = .
 
+supervisor-client-startsecs = 10
+supervisor-client-stopwaitsecs = 5
+
+supervisor-memmon-instance-size = 1200MB
+supervisor-memmon-options = -p instance1=${buildout:supervisor-memmon-instance-size} -p instance2=${buildout:supervisor-memmon-instance-size} -p instance3=${buildout:supervisor-memmon-instance-size} -p instance4=${buildout:supervisor-memmon-instance-size} -m ${buildout:supervisor-email}
+
+parts -= gems
+sablon-executable = /opt/sablon/bin/sablon
```
Comment (njohner) [inline opengever/policytemplates/policy_template/opengever.+package.name+/deployment-+base.server_name+-+base.deployment_number+-+base.domain+.cfg.bob:29]: Hum, I've actually never seen these two lines in any policy... is this correct?

Comment (elioschmutz) [reply opengever/policytemplates/policy_template/opengever.+package.name+/deployment-+base.server_name+-+base.deployment_number+-+base.domain+.cfg.bob:29]: It's currently used only in the `fmh` deployment: https://github.com/4teamwork/opengever.fmh/blob/master/deployment-fmh46.fmh.fmh-04-geverint.fmh.ch-int.cfg#L13

According to @buchi it should be de default...

@buchi is this correct?

Comment (buchi) [reply opengever/policytemplates/policy_template/opengever.+package.name+/deployment-+base.server_name+-+base.deployment_number+-+base.domain+.cfg.bob:29]: Absolutely! We install sablon with an RPM package on the server under the path `/opt/sablon/bin/sablon`. Installing it with buildout just slows down buildout.

Comment (deiferni) [reply opengever/policytemplates/policy_template/opengever.+package.name+/deployment-+base.server_name+-+base.deployment_number+-+base.domain+.cfg.bob:29]: please adapt all policies at once in an orderly manner then and get rid of the parts `-=` hack. it is no use if the sablom rpm is installed but we are unaware. separate jira issue i presume?

Comment (elioschmutz) [reply opengever/policytemplates/policy_template/opengever.+package.name+/deployment-+base.server_name+-+base.deployment_number+-+base.domain+.cfg.bob:29]: @deiferni 👍 I'll create a new jira story for that

Comment (elioschmutz) [reply opengever/policytemplates/policy_template/opengever.+package.name+/deployment-+base.server_name+-+base.deployment_number+-+base.domain+.cfg.bob:29]: @deiferni https://4teamwork.atlassian.net/browse/GEVER-485

**Pull request merged**